specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3556,9 +3556,90 @@
         "clientId"
       ]
     },
+    "FlexProfile": {
+      "type": "object",
+      "properties": {
+        "scaleProfile": {
+          "$ref": "#/definitions/ScaleProfile",
+          "description": "Specifications on how to scale a Flex agent pool."
+        }
+      },
+      "description": "Specifications on Flex agent pool."
+    },
+    "ScaleProfile": {
+      "type": "object",
+      "properties": {
+        "manual": {
+          "$ref": "#/definitions/ManualScaleProfile",
+          "description": "Specifications on how to scale the Flex agent pool to a fixed size."
+        }
+      },
+      "description": "Specifications on how to scale a Flex agent pool."
+    },
+    "ManualScaleProfile": {
+      "type": "object",
+      "properties": {
+        "machines": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManualScaleMachines"
+          },
+          "description": "Specifications on how to scale a Flex agent pool."
+        }
+      }
+    },
+    "ManualScaleMachines": {
+      "type": "object",
+      "properties": {
+        "sizes": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
Is this actually VM sizes, or is it VM families? `STANDARD_DS2_V2` or `DS2`?

Reply:
We'll start with VM size - STANDARD_DS2_V2. I'll update the example to reflect this.